eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```
@@ -39,7 +39,9 @@ parameters:
 - name: PreGenerationSteps
   type: stepList
   default: []
-
+- name: ProjectListOverrideFile
```

```suggestion
- name: ProjectListOverrideFileProperty
```

I think we want this to be empty by default

---

Done